src/app/project.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { of } from 'rxjs/observable/of';
+import { catchError, map, tap } from 'rxjs/operators';
+
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { Project } from './models/project';
+
+const httpOptions = {
```

Comment:
would be cool to encaspulate this outside, or do it in  service configuration code. a class or interceptor

Reply:
Agreed! Will tackle that in future refactoring.